### PR TITLE
feat(Kommander): project addon catalog

### PR DESCRIPTION
**Pull request by brandonc**
Closes D2IQ-64595

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-64595

## Description of changes being made

Adds a subsection to Projects describing addon repositories and addons

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/projects/index.md [hunk 1]
```diff
@@ -46,4 +42,4 @@ Using project ConfigMaps and project Secrets you can define configuration resour
 
 ## Platform Services
 
-Platform Services provide a catalog of applications that can be deployed across all project clusters.
+Platform Services provide a catalog of applications that can be deployed across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/projects/index.md]:
```suggestion
Platform Services provide a catalog of applications that can be deployed across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+layout: layout.pug
+navigationTitle: Addon Catalog
+title: Addon Catalog
+menuWeight: 8
+excerpt: Deploy applications to your Kommander Projects
+---
+
+Kommander includes the ability to deploy services from a catalog of current cloud native services with the push of a button. This helps organizations quickly deploy services to multiple clusters. Governance-based access to data services ensures roles and responsibilities are maintained.
+
+Kommander can be extended with new AddonRepository that point to git repositories containing application addons. For example, the kubeaddons-enterprise repo contains addons such as Jenkins and Kafka with specific settings for each cloud provider. Addons can be composed using either Helm V2 charts or KUDO operators.
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
Kommander can be extended with the `AddonRepository` resource that point to git repositories containing application addons. For example, the `kubeaddons-enterprise` repo contains addons such as Jenkins and Kafka with specific settings for each cloud provider. Addons can be composed using either Helm V2 charts or KUDO operators.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Example `AddonRepository` resource to add a new repository to your catalog
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
Example `AddonRepository` resource to add a new repository to your catalog:
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md (hunk 2 above)
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
Kommander can be extended with the new `AddonRepository` that points to git repositories containing application addons. For example, the kubeaddons-enterprise repo contains addons such as Jenkins and Kafka, with specific settings for each cloud provider. Addons can be composed using either Helm V2 charts or KUDO operators.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+metadata:
+  name: kubeaddons-enterprise
+  namespace: project-namespace
+spec:
+  priority: "20"
+  ref: “stable-1.16”
+  url: https://github.com/mesosphere/kubeaddons-enterprise
+```
+
+To deploy an addon, select a Project from your Workspace and click the View Catalog button to browse the available addons from your configured repositories.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
To deploy an addon, select a Project from your Workspace and select the **View Catalog** button to browse the available addons from your configured repositories.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+To customize an addon before deploying it, you may customize the helm chart values or KUDO parameters of a service before deploying it.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
To customize an addon before deploying it, you can customize the helm chart values or KUDO parameters of a service before deploying it.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+![Deploy Platform Service](/ksphere/kommander/img/project-catalog-deploy.png)
+
+Once an addon is deployed in a project, the service is installed to all clusters in that project.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
After an addon is deployed in a project, the service is installed to all clusters in that project.
```

File: pages/ksphere/kommander/latest/projects/index.md (hunk 1 above)
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/index.md]:
```suggestion
Platform Services provide a catalog of applications you can deploy across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+---
+layout: layout.pug
+navigationTitle: Addon Catalog
+title: Addon Catalog
+menuWeight: 8
+excerpt: Deploy applications to your Kommander Projects
+---
+
+Kommander includes the ability to deploy services from a catalog of current cloud native services with the push of a button. This helps organizations quickly deploy services to multiple clusters. Governance-based access to data services ensures roles and responsibilities are maintained.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md]:
```suggestion
Kommander can also deploy services from a catalog of current cloud native services with the push of a button. This helps organizations quickly deploy services to multiple clusters. Governance-based access to data services ensures roles and responsibilities are maintained.
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md (hunk 2 above)
Comment (danielone) [reply pages/ksphere/kommander/latest/projects/addon-catalog/index.md]: @brandonc would mention "platform services" here / add qualifier

**Pull request merged**